config.py

```diff
@@ -108,6 +106,15 @@ class Config(object):
     LABELS_GET_API_URL = RAS_SECURE_MESSAGE_SERVICE + 'labels?name=unread'
     THREAD_GET_API_URL = RAS_SECURE_MESSAGE_SERVICE + 'thread/{}'
 
+    RAS_FRONTSTAGE_API_HOST = os.getenv('RAS_FRONTSTAGE_API_HOST', 'localhost')
+    RAS_FRONTSTAGE_API_PORT = os.getenv('RAS_FRONTSTAGE_API_PORT', 8083)
+    RAS_FRONTSTAGE_API_PROTOCOL = os.getenv('RAS_FRONTSTAGE_API_PROTOCOL', 'http')
+    RAS_FRONTSTAGE_API_SERVICE = '{}://{}:{}/'.format(RAS_FRONTSTAGE_API_PROTOCOL,
+                                                      RAS_FRONTSTAGE_API_HOST,
+                                                      RAS_FRONTSTAGE_API_PORT)
+    GET_MESSAGES_URL = 'messages_list'
+    UNREAD_MESSAGES_TOTAL_URL = 'unread_message_total'
```

Comment:
Is this used?

Reply:
Nope this got absorbed into the frontstage api so I'll remove